test/functional/fixtures/runner/browser-dialogs/before-unload-dialog-test.js

```diff
@@ -0,0 +1,34 @@
+var expect = require('chai').expect;
+
+describe('Native dialogs handling', function () {
+    it('Should succeed when expected before unload dialog appeared', function () {
+        return runTests('before-unload-dialog.test.js', 'Before unload dialog - should succeed')
+            .then(function (err) {
+                expect(err).eql('');
+            });
+    });
+
+    it('Should fail when expected before unload dialog did not appear', function () {
+        return runTests('before-unload-dialog.test.js', 'No expected before unload dialog - should be failed', { shouldFail: true })
+            .catch(function (err) {
+                var expectedError = [
+                    'Error at step "1.Click link "This page"":',
+                    ' The expected system beforeUnload dialog did not appear.'
```

Comment:
@MargaritaLoseva This error template was reviewed before when it was commited, It's defined [here](https://github.com/DevExpress/testcafe/blob/master/src/reporter/errors/templates.js#L209).  @AndreyBelym just use it here. So, I propose to don't change it in this pr. If you would like to correct them we can create a separate issue to discuss this.